lib/local-links-manager/import/service_interactions_importer.rb

```diff
@@ -70,6 +81,14 @@ def create_or_update_record(parsed_hash)
 
         service_interaction.save!
       end
+
+      def import_summary
+        "ServiceInteraction Import complete\n"\
+        "Downloaded CSV rows: #{downloaded_csv_rows.count}\n"\
+        "Imported records: #{ServiceInteraction.count}\n"\
```

Comment:
Thank you, have addressed your comment. I went with another count variable for successful rows added, mainly because I'm not 100% sure that the code catches all errors (there might be more than missing ids and missing associated records)